features/polyExplorer/src/components/baseInfoScreen/baseInfoScreen.jsx

```diff
@@ -6,23 +6,27 @@ import Screen from "../screen/screen.jsx";
 
 import "./baseInfoScreen.css";
 
-const InfoScreen = ({ className, headline, children }) => {
+const BaseInfoScreen = ({ className, infoChildren }) => {
     const { handleBack } = useContext(ExplorerContext);
     return (
         <Screen className={className} light={true}>
             <div className="base-info">
-                <div className="base-info-padding">
-                    <h1>{headline}</h1>
-                </div>
-                {children}
-                <div className="base-info-padding">
-                    <button onClick={() => handleBack()}>
-                        {i18n.t("common:button.ok")}
-                    </button>
-                </div>
+                {infoChildren.map((infoChild, i) => (
+                    <div key={i}>
+                        <div className="info-title">
+                            <div className="line"></div>
+                            <h1>{i18n.t(`baseInfoScreen:title${i + 1}`)}</h1>
+                            <div className="line"></div>
+                        </div>
```

Comment:
Are you planning to add the SectionTitle-component here or do you want to keep it for later or me do it? Anyways we have to add a condition to the section title for the font-size (HTRT-titles are much smaller than the section titles – or is this the reason why you didn't use it, because HTRT-title is not really a section title?)